client/service_inspect.go

```diff
@@ -3,16 +3,20 @@ package client
 import (
 	"bytes"
 	"encoding/json"
+	"fmt"
 	"io/ioutil"
 	"net/http"
+	"net/url"
 
 	"github.com/docker/docker/api/types/swarm"
 	"golang.org/x/net/context"
 )
 
 // ServiceInspectWithRaw returns the service information and the raw data.
-func (cli *Client) ServiceInspectWithRaw(ctx context.Context, serviceID string) (swarm.Service, []byte, error) {
-	serverResp, err := cli.get(ctx, "/services/"+serviceID, nil, nil)
+func (cli *Client) ServiceInspectWithRaw(ctx context.Context, serviceID string, insertDefaults bool) (swarm.Service, []byte, error) {
```

Comment:
SGTM. I noticed that there was a mix of styles, with some functions taking `verbose`, or `force` boolean arguments. I can switch to an options structure if that's the preferred style.